### ISSUE TITLE
Rename battery-widget.lua to init.lua

### ISSUE THREAD

**Issue by Informatic** [coldfix/awesome.battery-widget#6]
Hey.
Renaming battery-widget.lua to init.lua would make using Your module slightly cleaner in dotfiles-repository-like environments.
By default `require('MODULE')` will try to load `MODULE.lua`, and then `MODULE/init.lua`, thus this repo could be easily used as a submodule in git and such, without playing with symlinks.

**Comment by coldfix** [comment]
Good idea. Is it fine, if I just add a symlink `init.lua -> battery-widget.lua` to the repo?

**Comment by Informatic** [comment]
I think that'd work as well, thanks!

**Comment by coldfix** [comment]
Seems to work. Thanks.

BTW, I was recently thinking about allowing installation via `luarocks`. What do you think?

**Comment by Informatic** [comment]
TBH I'm not much of a Lua guy, and I don't really think average awesomewm user is either. I think using submodules/simple cloning is a much simpler/more user-friendly way of handling deps. (But having luarocks package as an option would be a nice thing as well!)

BTW. I'm playing with doing proper ACPI events monitoring, so that `battery:update()` is called right when an actual event occurs. On AwesomeWM 4 it should be easily doable with `awful.spawn.with_line_callback`, not sure if it has been present in previous releases, as there is no public docs for these anymore, but this works more-or-less:
```bash
bash -c 'acpi_listen | while read n ; do echo $n ; echo battery:update\(\) | awesome-client; done'
```
I'll upgrade to AwesomeWM 4 and maybe drop a pull request here as an option.

Also, please update README that this widget already is awesome4-only, as `gears.timer` is unavailable on awesome 3.5.6

**Comment by coldfix** [comment]
> TBH I'm not much of a Lua guy, and I don't really think average awesomewm user is either. I think using submodules/simple cloning is a much simpler/more user-friendly way of handling deps. (But having luarocks package as an option would be a nice thing as well!)

Makes sense. I meant as an alternative, of course. Since I have a couple of small plugins, I was also thinking about merging them into a single repo to share some code and reduce maintenance.

> I'll upgrade to AwesomeWM 4 and maybe drop a pull request here as an option.

That would be great.

> Also, please update README that this widget already is awesome4-only, as gears.timer is unavailable on awesome 3.5.6

Okay. Is that the only issue? In this case, I'll just fallback to `timer` for now. (`gears` itself is available on 3.5. right?)

**Comment by Informatic** [comment]
Yes, that's the only issue. `gears` is available, but missing `timer`.
